source/Engine.cpp

```diff
@@ -2504,13 +2510,14 @@ void Engine::DrawShipSprites(const Ship &ship)
 	bool hasFighters = ship.PositionFighters();
 	double cloak = ship.Cloaking();
 	bool drawCloaked = (cloak && ship.IsYours());
+	bool fancyCloak = Preferences::Has("Cloaked ship outlines");
 	auto &itemsToDraw = draw[currentCalcBuffer];
-	auto drawObject = [&itemsToDraw, cloak, drawCloaked](const Body &body) -> void
+	auto drawObject = [&itemsToDraw, cloak, drawCloaked, fancyCloak](const Body &body) -> void
 	{
-		// Draw cloaked/cloaking sprites swizzled red, and overlay this solid
-		// sprite with an increasingly transparent "regular" sprite.
+		// Draw cloaked/cloaking sprites swizzled red or transparent (depending on whether we are using fancy
+		// cloaking effects), and overlay this solid sprite with an increasingly transparent "regular" sprite.
 		if(drawCloaked)
-			itemsToDraw.AddSwizzled(body, 27);
+			itemsToDraw.AddSwizzled(body, fancyCloak ? 9 : 27, fancyCloak ? 0.5 : 0.25);
```

Comment:
Currently, cloaking doesn't provide any transparency. Why add 50% transparency to fast cloaking?
![image](https://github.com/endless-sky/endless-sky/assets/17688683/f4581b8f-06a6-48e3-849b-ff3085466286)
![image](https://github.com/endless-sky/endless-sky/assets/17688683/94414072-cd68-46a9-8c2e-f072fbae37ac)

Reply:
Maybe 25% is not enough. I think maybe 45% should be better.
![image](https://github.com/endless-sky/endless-sky/assets/87318892/947575da-13db-4199-8ff6-80f5f5ad73a7)
I think with the 35% of transparency and the red outline, it's looking good now.

But adding transparency is definitely a good idea.

Reply:
> Currently, cloaking doesn't provide any transparency. Why add 25% transparency to fast cloaking? ![image](https://private-user-images.githubusercontent.com/17688683/298256564-f4581b8f-06a6-48e3-849b-ff3085466286.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0NzMxNDcsIm5iZiI6MTcwODQ3Mjg0NywicGF0aCI6Ii8xNzY4ODY4My8yOTgyNTY1NjQtZjQ1ODFiOGYtMDZhNi00OGUzLTg0OWItZmYzMDg1NDY2Mjg2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIwVDIzNDcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkY2Q3MmI5N2ZjMzY0OTk2OWIwNTk0N2Q3ZjI2N2ZjMGEzNWE5MTVlZWIxYTlkMTFhZDY5MDNkZWQ0YWQyMGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uCfUjpqYLILlrmgOHGFPknM8NKLUdHos-PUEzL_9Amc) ![image](https://private-user-images.githubusercontent.com/17688683/298256589-94414072-cd68-46a9-8c2e-f072fbae37ac.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0NzMxNDcsIm5iZiI6MTcwODQ3Mjg0NywicGF0aCI6Ii8xNzY4ODY4My8yOTgyNTY1ODktOTQ0MTQwNzItY2Q2OC00NmE5LThjMmUtZjA3MmZiYWUzN2FjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIwVDIzNDcyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjNmY0YWQzZDkwZmYwMTZhMzgxOTAwZTQ1NDZkYjUyODcyMDI3Y2ZlZDE1Y2NkZjg1ZWQ0ODZjNDRhZGJlNjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qnQhRlHHgfN4llfu1F9P0LWmnytkbd1Bf_53OHjcmYw)

It looks cooler and is less harsh on the eyes

Reply:
Fair enough.